### DISCUSSION TITLE
求助！我下载并构建了作者的github项目但是我不知道应该如何运行

### DISCUSSION THREAD

**Discussion by yunzaiawa** [share121/inter-knot#2556] [其他]
![QQ20240816-233249](https://github.com/user-attachments/assets/9e067552-b5b0-4c06-aca0-4f7dec37a808)
没有start脚本！项目也没写运行方法！

**Comment by Pleasurecruise** [comment]
左边栏网站，选node项目，配置一下端口，启动项，版本。这个项目我没试过，本地可以跑。

**Comment by HifumiMizuhara** [comment]
```shell
npm install
npm run dev
```

**Reply by HifumiMizuhara** [reply]
这就是个 Nuxt 应用，用 Nuxt 固有的方式就可以打开。
